query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/CswSpatialOperator.java

```diff
@@ -51,209 +50,281 @@ public CswSpatialOperator(DataType<String> spatialOperator) {
                 spatialOperator);
     }
 
-    protected static final class NoFilter extends StringField {
+    public static final class NoFilter implements EnumValue<String> {
 
-        public static final String OPERATOR = "NO_FILTER";
+        public static final String ENUM_TITLE = "None";
 
-        public static final String NO_FILTER_FIELD_NAME = "None";
+        public static final String OPERATOR = "NO_FILTER";
 
         public static final String DESCRIPTION =
                 "Indicates that no spatial operators should be applied.";
 
-        public NoFilter() {
-            super(NO_FILTER_FIELD_NAME, NO_FILTER_FIELD_NAME, DESCRIPTION);
+        @Override
+        public String enumTitle() {
+            return ENUM_TITLE;
```

Comment:
Well, I finally encountered an enum with different values for `value()` and `enumTitle()`. If it's this rare, then it might be better to change from an interface to an abstract base class and provide a default implementation of `value()` that returns `enumTitle()`. Of course, I also now see looking closer that `value()` returns type `T`...but do you actually have or foresee any enums of types _other_ than String?